go/util/profile/profile.go

```diff
@@ -30,6 +35,25 @@ func RegisterProfileFlags(flags *flag.FlagSet) {
 	}
 }
 
+func KAddProfileFlags(cmd *kingpin.CmdClause) {
```

Comment:
Same thing with the naming here, the `k` prefix is confusing. Can we just drop it?